.devcontainer/Dockerfile

```
@@ -0,0 +1,34 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.1.602
```

Master updated to `2.1.802`, I'll just quickly update this
```suggestion
FROM mcr.microsoft.com/dotnet/core/sdk:2.1.802
```